backend/tests/ingestion/test_pipeline.py

```
@@ -0,0 +1,105 @@
+            s3_bucket_name="test-bucket",
+            client=mock_crlca_client
+        )
+
+@pytest.mark.asyncio
+async def test_fetch_and_normalize_case_supported_document(pipeline, mock_crlca_client, mock_s3_client):
+    # Setup mocks
+    mock_crlca_client.fetch_case_details.return_value = {
+        "id": 1, "name": "Test Case", "court": "Federal Court"
+    }
+
+    async def mock_dockets(*args, **kwargs):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `mock_dockets`

(ANN202)